generator/tests/test_other_library/test-code/omex/CaBase.cpp

```diff
@@ -2173,7 +2173,7 @@ CaBase::read (LIBSBML_CPP_NAMESPACE_QUALIFIER XMLInputStream& stream)
         {
           unsigned int errorId =
                              this->getErrorLog()->getError(n)->getErrorId();
-          if ( errorId == CombinemlOmexManifestAllowedCoreAttributes
+if ( errorId == CombinemlOmexManifestAllowedCoreAttributes
```

Comment:
I don't think the indentation for this if-statement is correct.

Reply:
No its not but I left it so tests pass because I need to revisit all the other_library stuff and know I will be changing tests

Reply:
I cheated with some changes and just copied the whole file - this was more about getting the test_other_library tests to run